.travis.yml

```diff
@@ -0,0 +1,41 @@
+# R for travis: see documentation at https://docs.travis-ci.com/user/languages/r
+
+language: R
+warnings_are_errors: false
+
+matrix:
+  include:
+    - os: linux
+      cache: packages
+      dist: precise
+      sudo: false
+      r: release
+      addons:
+        apt:
+          sources:
+            - ubuntu-toolchain-r-test
+          packages:
+            - gcc-4.9
+            - g++-4.9
+            - iodbc
+            - libiodbc2
+            - libiodbc2-dev
+      before_install: |
+        mkdir ~/.R
+        cat <<EOF > ~/.R/Makevars
+        CXX1X=g++-4.9
+        CXX1XSTD=-std=c++11
+
+    - os: osx
+      osx_image: xcode8.2
+      sudo: required
+      r: release
+      before_install:
+        - brew update
+      install:
+        - sudo chown -R $(whoami) /usr/local/share/man/man7
+        - brew install unixodbc
+        - Rscript -e 'install.packages("devtools", repos = "http://cran.rstudio.com")'
+        - Rscript -e 'install.packages(c("caret","data.table","doParallel","e1071"), repos = "http://cran.rstudio.com")'
+        - Rscript -e 'install.packages(c("grpreg","lme4","lubridate","pROC"), repos = "http://cran.rstudio.com")'
+        - Rscript -e 'install.packages(c("ranger","ROCR","testthat","RODBC"), repos = "http://cran.rstudio.com")'
```

Comment:
I'm not sure why this didn't work as you describe. Travis was unable to find devtools. Added line 38, and travis couldn't find any of the other packages. It only seemed to work with them installed manually.